README.md

```diff
@@ -15,6 +15,9 @@ This is a project template for [vue-cli](https://github.com/vuejs/vue-cli). **It
 
 ``` bash
 $ npm install -g vue-cli
+# the master branch now is working with vue 2
```

Comment:
since this is already mentioned in Line 5 of the README, this should be removed.